native/cocos/bindings/jswrapper/v8/ScriptEngine.h

```diff
@@ -317,6 +317,16 @@ class ScriptEngine final {
      */
     uint32_t getVMId() const { return _vmId; }
 
+    /**
+     * @brief
+     * This function asks ScriptEngine to interrupt lengthy JavaScript operations and run the provided callback, passing the supplied data to it.
+     * Once the callback completes, control returns to the JavaScript code. Multiple interrupt requests may be ongoing.
+     * This function can be called from a different thread without needing a Locker. The registered callback should not reenter the interrupted Isolate.
+     * @param callback
+     * @param data
```

Comment:
Missing comments for callback and data arguments.

Reply:
Updated